apps/src/applab/applab.js

```diff
@@ -80,6 +80,21 @@ export default Applab;
 var jsInterpreterLogger = null;
 
 /**
+ * Whether Maker Toolkit is enabled on this level.  Will be initialized on
+ * level load.
+ * @private {boolean}
+ */
+let makerEnabled = false;
+
+/**
+ * Maker Toolkit Board Controller for a currently-connected board, simulator,
+ * or stub implementation.
+ * In Maker Toolkit levels, should be initialized on run and cleared on reset.
+ * @private {CircuitPlaygroundBoard}
+ */
+let makerBoard = null;
```

Comment:
Doing this, and removing `Applab.makerController`, to ensure that this reference is private.